modules/core/shared/src/main/scala/gem/enum/GsaoiFilter.scala

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GSAOI Filter.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GsaoiFilter(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val wavelength: Double,
+  val readMode: GsaoiReadMode,
+  val expsoureTime5050: Double,
```

Comment:
Same here, but for `Duration`.  At the moment we're using Java `Duration` so I suppose it should be that but we'll probably move to a Scala `FiniteDuration` at some point.